tests-integration/src/lib.rs

```diff
@@ -0,0 +1 @@
+doc_comment::doc_comment!(include_str!("../README.md"));
```

Comment:
I think, you meant to use:
```
#![doc = include_str!("../README.md")]
```

Ref: [doc attribute](https://doc.rust-lang.org/rustdoc/write-documentation/the-doc-attribute.html#the-doc-attribute)

Reply:
[58c6a24](https://github.com/cosmos/ibc-rs/pull/1233/commits/58c6a2420bf8d6dd673ae7b646332993fcaf0871)